docs/alerts-and-notifications/notifications/centralized-cloud-notifications/manage-notification-methods.md

```diff
@@ -63,7 +63,7 @@ Note: If an administrator has disabled a Personal [service level](/docs/alerts-a
 2. You are presented with:
    - The Personal [service level](/docs/alerts-and-notifications/notifications/centralized-cloud-notifications/centralized-cloud-notifications-reference.md#service-level) notification methods you can manage.
    - The list of Spaces and Rooms inside those where you have access to.
-   - If you're an Administrator, Manager, or Troubleshooter, you'll also see the Rooms from a Space you don't have access to on the **All Rooms** tab, and you can activate notifications for them by joining the Room.
+   - If you're an Administrator, Manager, you'll also see the Rooms from a Space you don't have access to on the **All Rooms** tab, and you can activate notifications for them by joining the Room.
```

Comment:
```suggestion
   - If you're an Administrator or Manager, the **All Rooms** tab will include Rooms from Spaces you don't typically have access to. This happens because of your administrative role. You can still join these Rooms to receive notifications for them.
```

@Ancairon 

- what is **All Rooms** tab?
- "ooms from Spaces you don't typically have access to" ?

Reply:
whoever wrote this I think wanted to indicate the all rooms button on the left bar

Reply:
>- If you're an Administrator or Manager, the **All Rooms** tab will include Rooms from Spaces you don't typically have access to. This happens because of your administrative role. You can still join these Rooms to receive notifications for them.

maybe `will include Rooms from the Space where you don't typically have access to.`?

Reply:
as you write it I understand "from other Spaces"

Reply:
This sentence doesn't make sense to me. Let's remove it.

Reply:
removed it.